baremaps-geoparquet/src/main/java/org/apache/baremaps/geoparquet/data/GeoParquetGroupImpl.java

```diff
@@ -414,6 +427,22 @@ public List<Geometry> getGeometryValues(int fieldIndex) {
     return geometries;
   }
 
+  @Override
+  public Envelope getEnvelopeValue(int fieldIndex) {
+    return getEnvelopeValues(fieldIndex).get(0);
+  }
+
+  @Override
+  public List<Envelope> getEnvelopeValues(int fieldIndex) {
+    return getGroupValues(fieldIndex).stream().map(group -> {
+      var xMin = group.getDoubleValue(0);
+      var yMin = group.getDoubleValue(1);
+      var xMax = group.getDoubleValue(2);
+      var yMax = group.getDoubleValue(3);
```

Comment:
With a Overture maps file this fails because the bbox uses float values.

```suggestion
      var xMin = group.getFloatValue(0);
      var yMin = group.getFloatValue(1);
      var xMax = group.getFloatValue(2);
      var yMax = group.getFloatValue(3);
```